docs/docs/fundamentals/about.md

```diff
@@ -6,15 +6,15 @@ sidebar_label: About
 
 :::info
 
-This is a documentation website of Reanimated 2 stable release.
+This is a documentation website of Reanimated 3.x release.
 
-If you are looking for Reanimated 1 docs [please follow this link](https://docs.swmansion.com/react-native-reanimated/docs/1.x/).
+Looking for an older version? Check the [Reanimated 2.x](/docs/2.x/) or [Reanimated 1.x](/docs/1.x/).
 
 :::
 
 :::caution
 
-Reanimated v2 only supports `react-native` `0.62+`
+Reanimated 3.x aims to support [3 last stable React Native versions](https://github.com/reactwg/react-native-releases#which-versions-are-currently-supported).
```

Comment:
Reanimated 3.0.0 aims to support RN 0.64+ but we don't make such promise for the next releases.

Reply:
Fixed in [175b95f](https://github.com/software-mansion/react-native-reanimated/pull/4102/commits/175b95faacec0747737d821c760e9b087cbeee8d)